src/Pages/Login/Login.js

```diff
@@ -0,0 +1,103 @@
+import { useNavigate } from "react-router-dom";
+import { useToast } from "../../Context/ToastContext";
+import { useState } from "react";
+
+import { loginHandle } from "../../FirebaseCall/loginHandle";
+// import { userLogin } from "../../ApiCalls";
+
+export const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [isloading, setIsloading] = useState(false);
```

Comment:
Suggestion: You can use global state management for this so that you can get loader status in different components

Reply:
Yaa... I could done that..
But it is small app , so i sticked with this..